egs/ifnenit/v1/local/reduce-to-vocabulary.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
```

Comment:
This and other scripts in local/ need documentation. Please use underscores in the file name.